source/GlassView.Export/Export.cs

```diff
@@ -0,0 +1,97 @@
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Mathematics;
+using BenchmarkDotNet.Portability.Cpu;
+using BenchmarkDotNet.Reports;
+using GlassView.Core.Models;
+
+namespace GlassView.Export;
+
+public static class Export
```

Comment:
Maybe rename the class to "BenchmarkDeserializer" or "BenchmarkMapper" because it sort of deserializes the summary to our own data.

I would also rename the public parameter "summary" to "inputSummary" 😊

Reply:
I've done some renaming. Let's look at the types in a 1:1 call sometime next week :-)

Reply:
Should we have our call on Friday morning? My program seems to have a little room to wiggle right now 😁